Bloop console exits with NoSuchMethodError

I'm using bloop-1.4.1 (installed via Coursier) and receiving an error when trying to issue `bloop console <project>`.

```
$ bloop about
bloop v1.4.1

Using Scala v2.12.8 and Zinc v1.3.0-M4+42-5daa8ed7
Running on Java JDK v1.8.0_222 (/nix/store/a6nkkh7fwjz6bl6im35jq77fibh85wrn-openjdk-8u222-ga/lib/openjdk/jre)
  -> Supports debugging user code, Java Debug Interface (JDI) is available.
Maintained by the Scala Center (Jorge Vicente Cantero, Martin Duhem)
```
```
$ bloop console <project>
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Duser.timezone=UTC
Exception in thread "main" java.lang.NoSuchMethodError: scala.collection.immutable.Map.map(Lscala/Function1;)Lscala/collection/IterableOps;
	at ammonite.main.ProxyFromEnv$.getEnvs(ProxyFromEnv.scala:23)
	at ammonite.main.ProxyFromEnv$.setPropProxyFromEnv$default$1(ProxyFromEnv.scala:54)
	at ammonite.Main$.main(Main.scala:268)
	at ammonite.Main.main(Main.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at coursier.cli.launch.Launch$.$anonfun$launch$6(Launch.scala:174)
	at coursier.cli.launch.Launch$.$anonfun$launchCall$11(Launch.scala:329)
	at coursier.cli.launch.Launch$.run(Launch.scala:515)
	at coursier.cli.Coursier$.$anonfun$runA$2(Coursier.scala:127)
	at coursier.cli.Coursier$.$anonfun$runA$2$adapted(Coursier.scala:113)
	at coursier.cli.CommandAppPreA.run(CommandAppPreA.scala:22)
	at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$5(CommandAppWithPreCommand.scala:99)
	at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$5$adapted(CommandAppWithPreCommand.scala:99)
	at scala.util.Either.fold(Either.scala:191)
	at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$3(CommandAppWithPreCommand.scala:99)
	at caseapp.core.app.CommandAppWithPreCommand.$anonfun$main$3$adapted(CommandAppWithPreCommand.scala:85)
	at scala.Option.foreach(Option.scala:407)
	at caseapp.core.app.CommandAppWithPreCommand.main(CommandAppWithPreCommand.scala:85)
	at coursier.cli.Coursier$.main(Coursier.scala:78)
	at coursier.cli.Coursier.main(Coursier.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at coursier.bootstrap.launcher.a.a(Unknown Source)
	at coursier.bootstrap.launcher.Launcher.main(Unknown Source)
```

I've narrowed this down a bit. I've shortened the command generated by `bloop console` removing the `--extra-jars` args, while reproducing the issue. Then I found that dropping the `--scala-version 2.13.2` arg seems to work.

Not sure how to proceed in terms of a patch, but hopefully someone else can do that easily.

```
bloop git:(master) ✗ coursier launch com.lihaoyi:ammonite_2.13.2:latest.release --main-class ammonite.Main --scala-version 2.13.2 
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Duser.timezone=UTC
Exception in thread "coursier-pool-1-thread-6" java.lang.NoClassDefFoundError: scala/collection/IterableOnce
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at coursier.cli.launch.Launch$.liftedTree2$1(Launch.scala:150)
	at coursier.cli.launch.Launch$.$anonfun$launch$3(Launch.scala:149)
	at scala.util.Either.flatMap(Either.scala:341)
	at coursier.cli.launch.Launch$.launch(Launch.scala:141)
	at coursier.cli.launch.Launch$.launchCall(Launch.scala:328)
	at coursier.cli.launch.Launch$.$anonfun$task$7(Launch.scala:452)
	at coursier.cli.launch.Launch$.$anonfun$task$7$adapted(Launch.scala:432)
	at coursier.util.Task$.$anonfun$flatMap$2(Task.scala:14)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: scala.collection.IterableOnce
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	... 19 more
^C%                                                            
                                                                                                                                                                                                                                                                bloop git:(master) ✗ coursier launch com.lihaoyi:ammonite_2.13.2:latest.release --main-class ammonite.Main                       
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Duser.timezone=UTC
Loading...
Welcome to the Ammonite Repl 2.1.4-7-d1e71b9 (Scala 2.13.2 Java 1.8.0_242)
@  
```

--- a/frontend/src/main/scala/bloop/engine/Interpreter.scala
+++ b/frontend/src/main/scala/bloop/engine/Interpreter.scala
@@ -278,9 +278,7 @@ object Interpreter {
                     "launch",
                     s"com.lihaoyi:ammonite_$scalaVersion:$ammVersion",
                     "--main-class",
-                    "ammonite.Main",
-                    "--scala-version",
-                    scalaVersion
+                    "ammonite.Main"
                   )
 
                   val classpath = project.fullRuntimeClasspath(dag, state.client)

--- a/frontend/src/test/scala/bloop/ConsoleSpec.scala
+++ b/frontend/src/test/scala/bloop/ConsoleSpec.scala
@@ -43,7 +43,7 @@ object ConsoleSpec extends BaseSuite {
       val coursierClasspathArgs =
         classpathB.flatMap(elem => Seq("--extra-jars", elem.syntax))
       val expectedCommand =
-        s"coursier launch com.lihaoyi:ammonite_2.12.8:latest.release --main-class ammonite.Main --scala-version 2.12.8 ${coursierClasspathArgs
+        s"coursier launch com.lihaoyi:ammonite_2.12.8:latest.release --main-class ammonite.Main ${coursierClasspathArgs
           .mkString(" ")} ${("--" :: ammArgs).mkString(" ")}"
 
       assertNoDiff(

